### PR TITLE
 git-secret: init at 0.2.2

### PR DESCRIPTION
**Pull request by lo1tuma**
###### Motivation for this change

[git-secret](http://git-secret.io/) is a bash tool to store your private data inside a git repo.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/version-management/git-and-tools/git-secret/default.nix [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    wrapProgram $out/bin/git-secret \
+      --prefix PATH : "${lib.makeBinPath [ git gnupg ]}"
+
+    cp -R man $out/
+  '';
+
+  meta = {
+    description = "A bash-tool to store your private data inside a git repository.";
+    homepage = http://git-secret.io;
+    license = lib.licenses.mit;
```
Comment (yegortimoshenko) [inline pkgs/applications/version-management/git-and-tools/git-secret/default.nix]: Typically `stdenv.lib` is used instead.

File: pkgs/applications/version-management/git-and-tools/git-secret/default.nix [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{ pkgs, stdenv, lib, fetchFromGitHub, makeWrapper, git, gnupg }:
```
Comment (yegortimoshenko) [inline pkgs/applications/version-management/git-and-tools/git-secret/default.nix]: `pkgs` is not used in this derivation.

**Review by lukateras — APPROVED** [review]
Builds and works!

File: pkgs/applications/version-management/git-and-tools/git-secret/default.nix (hunk 2 above)
Comment (lo1tuma) [reply pkgs/applications/version-management/git-and-tools/git-secret/default.nix]: Good catch, removed.

File: pkgs/applications/version-management/git-and-tools/git-secret/default.nix (hunk 1 above)
Comment (lo1tuma) [reply pkgs/applications/version-management/git-and-tools/git-secret/default.nix]: Fixed.

**Pull request merged**